use-cases/vehicle-routing/src/test/java/org/acme/vehiclerouting/rest/VehicleRoutePlanResourceTest.java

```diff
@@ -22,6 +22,40 @@ public class VehicleRoutePlanResourceTest {
 
     @Test
     public void solveDemoDataUntilFeasible() {
+        VehicleRoutePlan solution = solveDemoData();
+        assertTrue(solution.getScore().isFeasible());
+    }
+
+    @Test
+    void analyze() {
```

Comment:
Maybe split into two test methods, one of them does "shallow fetch".

Reply:
I have copied the test structure from School timetabling QS for start, good point that it could be refactored, will take a look.

Reply:
Done.